src/cryo_challenge/_preprocessing/preprocessing_pipeline.py

```diff
@@ -107,6 +107,11 @@ def preprocess_submissions(submission_dataset, config):
         print("    Centering submission")
         volumes = center_submission(volumes, pixel_size=pixel_size_gt)
 
+        # flip handedness
+        if submission_dataset.submission_config[str(idx)]["flip"] == 1:
+            print("    Flipping handedness of submission")
+            volumes = volumes.flip(-1)
```

Comment:
in the docs (notebook?) you can explain what axis (the last ) that it flips over